common/taskana-common/src/main/resources/sql/postgres/taskana_schema_update_7.0.0_to_7.0.1_postgres.sql

```diff
@@ -0,0 +1,16 @@
+-- this script updates the TASKANA database schema from version 7.0.0 to version 7.0.1.
+
+SET search_path = %schemaName%;
+
+INSERT INTO TASKANA_SCHEMA_VERSION (ID, VERSION, CREATED)
+VALUES (nextval('TASKANA_SCHEMA_VERSION_ID_SEQ'), '7.0.1', CURRENT_TIMESTAMP);
+
+ALTER TABLE TASK
+    ADD COLUMN NUMBER_OF_COMMENTS INT4 DEFAULT 0;
```

Comment:
Why do you use INT4 as type here? Why is the type here different from the type in taskana-schema-postgres?

Reply:
Because in DBeaver it said INT4 next to the column name of all INT columns. I will change it to INT.